Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -3125,6 +3134,22 @@ private static bool HandleKeywords(Material destMat, string keyword)
 #if AT_USE_SPLINES
     PinnedList<float3> m_tmpFloat3 = new PinnedList<float3>();
 #endif
+    
+#if UNITY_EDITOR
+    [NonSerialized]
+    private List<RenderTexture> CreatedRenderTextures = new List<RenderTexture>();
+
+    public void AddCreatedRenderTexture(RenderTexture renderTexture) {
```

Comment:
Do we need to make this into a public function ?

Reply:
Made it internal